tests/core/non_linear_test_system_01.h

```diff
@@ -128,7 +114,44 @@ class TestClass : public PhysicsSolver<Vector<double>>
   apply_constraints()
   {}
 
+  virtual Vector<double> &
+  get_evaluation_point() override
+  {
+    return evaluation_point;
+  };
+  virtual Vector<double> &
+  get_local_evaluation_point() override
+  {
+    return local_evaluation_point;
+  };
+  virtual Vector<double> &
+  get_newton_update() override
+  {
+    return newton_update;
+  };
+  virtual Vector<double> &
+  get_present_solution() override
+  {
+    return present_solution;
+  };
+  virtual Vector<double> &
+  get_system_rhs() override
+  {
+    return system_rhs;
+  };
+  virtual AffineConstraints<double> &
+  get_nonzero_constraints() override
+  {
+    return dummy_constraints;
```

Comment:
is it a temporary name?

Reply:
It's actually an empty constraint. It is never used. So it is really a dummy constraint.